src/lazy/components/server.tsx

```diff
@@ -26,7 +23,7 @@ export const createComponentServer = ({
   loader: ServerLoader;
   ssr: boolean;
 }) => (props: any) => {
-  const Resolved = ssr ? tryRequire(cacheId) || load(cacheId, loader) : null;
+  const Resolved = ssr ? load(cacheId, loader) : null;
```

Comment:
Still wonder if this ternary should be needed. I theory if `ssr` is `false` our babel plugin could strip the import out and replace it with `() => null`. 
That might be a nice perf improvement as will prevent webpack from scanning and importing that code in the first place

Reply:
That replacement is something that Jira already does. I can look into adding in this behaviour into the plugin in a separate PR.